lalrpop-test/src/allow_catch_all.lalrpop

```diff
@@ -0,0 +1,16 @@
+grammar;
+
+match {
+    _
+} else {
+    r"\w+"
+}
+
```

Comment:
As @Marwes noted, if we're going to add a test here, we should add a `mod allow_catch_all;` in `lib.rs` and some code that uses it. That said, I think it might be better to write one of the more targetd unit tests, like those found in [this module](https://github.com/lalrpop/lalrpop/blob/df6fd076565530a5dcce269f8a171b5d4d934d36/lalrpop/src/normalize/token_check/test.rs#L159-L164)

Reply:
Ok, I see your point.
I will remove the integration test and add a targeted unit test in `token_check/test.rs` ;-)